web/pageComponents/shared/Hero/LoopingVideo.tsx

```diff
@@ -1,10 +1,19 @@
-import { HLSPlayer } from '@components'
+import { HLSPlayer } from '../../../components/src/HLSPlayer'
 import { useSanityLoader } from '../../../lib/hooks/useSanityLoader'
 import styled from 'styled-components'
 import { LoopingVideoData, LoopingVideoRatio } from '../../../types'
+import dynamic from 'next/dynamic'
 
 const DEFAULT_MAX_WIDTH = 1920
 
+const DynamicHLSVideoComponent = dynamic<React.ComponentProps<typeof HLSPlayer>>(
```

Comment:
is it possible to move this into its own component and then call upon it?

Reply:
In documentation for next dynamic, its called like this and React documentation for lazy, which next/dynamic is built upon,  is "Call lazy outside your components to declare a lazy-loaded React component".
So i think it needs to be declared like this.